plugins/inventory/icinga2.py

```diff
@@ -0,0 +1,223 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2016 Guido Günther <agx@sigxcpu.org>, Daniel Lobato Garcia <dlobatog@redhat.com>
+# Copyright (c) 2018 Ansible Project
```

Comment:
@BongoEADGC6 you may perhaps have copied this from a nother module - shouldn't it be your name in there? Also, updated to 2021.

Reply:
Thanks for pointing this out.